pyapr/numerics/transform/morphological.py

```diff
@@ -67,18 +67,29 @@ def bottomhat(apr: pyapr.APR,
 
 
 def remove_small_holes(apr: pyapr.APR,
-                       parts: (pyapr.ShortParticles, pyapr.FloatParticles),
-                       min_volume: int = 200):
-
-    mask = parts < 1
-    cc_inverted = pyapr.ShortParticles()
+                       parts: (pyapr.ShortParticles, pyapr.LongParticles, pyapr.FloatParticles),
+                       max_volume: int = 200):
+
+    if isinstance(parts, pyapr.ShortParticles):
+        mask = parts < 1
+        cc_inverted = pyapr.ShortParticles()
+    elif isinstance(parts, pyapr.LongParticles):
+        mask = parts < 1
+        mask = np.array(mask).astype('uint16')
+        mask = pyapr.ShortParticles(mask)
+        cc_inverted = pyapr.LongParticles()
     pyapr.numerics.segmentation.connected_component(apr, mask, cc_inverted)
```

Comment:
Are we missing a case for the pyapr.FloutParticles space, as in that case cc_inverted and mask will not be defined?